From ce74f4e436e978bb2c9ac24378eed6d6502267ca Mon Sep 17 00:00:00 2001
From: Xianzhong <xianzhong.li@nxp.com>
Date: Thu, 24 Aug 2017 13:44:50 +0800
Subject: [PATCH 2438/5242] MGS-3224: arm64: dts: fsl-imx8qxp-mek: enable gpu

commit  6a59c1a365180640236cb63f3a40e94ec3af1512 from
https://source.codeaurora.org/external/imx/linux-imx.git

this patch enable gpu feature for i.mx8qxp mek board

Signed-off-by: Xianzhong <xianzhong.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts |    8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index 2eaa2c1..ad18609 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -477,3 +477,11 @@
 &dpu1 {
 	status = "okay";
 };
+
+&gpu_3d0 {
+	status = "okay";
+};
+
+&imx8_gpu_ss {
+	status = "okay";
+};
-- 
1.7.9.5

